docs/src/token-2022/status.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Project Status
+---
+
+The Token-2022 program is still under audit and not meant for full production use.
+In the meantime, all clusters have the latest program deployed **for testing and
+development purposes ONLY**.
+
+## Timeline
+
+Here is the general program timeline and rough ETAs:
+
+| Issue | ETA |
+| --- | --- |
+| Finish program | end of May |
+| Final audit | end of June |
+| Mainnet recommendation | middle of July |
+| Freeze program | Q1 2024 |
+
+More information: https://github.com/orgs/solana-labs/projects/34
+
+## Remaining items
+
+### Zero-knowledge proof split
```

Comment:
The zk support also requires the cluster be running 1.14 + enabled feature gates, worth calling that out in this section too?

Reply:
I had it and then removed it to avoid putting too much, but since you've called it out, I think that means it's needed :smile: any ETA for 1.14 that I can add?

Reply:
For this doc I think it's best to just call out the dependency on 1.14 without a specific eta, easier to maintain going forward.

but for an 1.14 ETA, soon? Would love to see mainnet on 1.14 by the end of May. There's no real blocker anymore, just perception issues

Reply:
Sounds good, done!